modules/cluster-logging-log-store-status-viewing.adoc

```diff
@@ -187,7 +187,7 @@ status:
        type:                  Unschedulable
 ----
 
-This status message indicates that your log store cluster does not have enough nodes to support your log store redundancy policy.
+The following status message indicates that your log store cluster does not have enough nodes to support your log store redundancy policy.
```

Comment:
the word "your' is used twice. can remove the 1st occurrence.. WDYT?

Reply:
Nice catch.